spec/caskinstaller_spec.rb

```diff
@@ -37,6 +41,15 @@ def do_install
         expect(Bundle).to receive(:system).with("brew", "cask", "install", "google-chrome").and_return(true)
         expect(do_install).to eq(true)
       end
+
+      it "works fine with arguments" do
+        allow(Bundle).to receive(:cask_installed?).and_return(false, true)
+        allow(ARGV).to receive(:verbose?).and_return(false)
+        allow(Bundle::CaskInstaller).to receive(:installed_casks).and_return([])
+        expect(Bundle).to receive(:system).with("brew", "install", "caskroom/cask/brew-cask").and_return(true)
+        expect(Bundle).to receive(:system).with("brew", "cask", "install", "firefox", "--appdir=/Applications").and_return(true)
+        expect(do_install_with_args).to eq(true)
+      end
```

Comment:
This test may be more suitable to put after [Line 90](https://github.com/Liquidsoul/homebrew-bundle/blob/master/spec/caskinstaller_spec.rb#L90)